packages/sit-onyx/src/components/OnyxLink/OnyxLink.vue

```diff
@@ -24,21 +26,20 @@ defineSlots<{
   default(): unknown;
 }>();
 
+const { t } = injectI18n();
+
 const shouldShowExternalIcon = computed(() => {
   if (props.withExternalIcon !== "auto") return props.withExternalIcon;
   return isExternalLink(props.href);
 });
 </script>
 
 <template>
-  <a
-    class="onyx-link"
-    :href="props.href"
-    :target="props.target"
-    :rel="props.target === '_blank' ? 'noreferrer' : undefined"
-    @click="emit('click')"
-  >
+  <a class="onyx-link" :href="props.href" :target="props.target" @click="emit('click')">
```

Comment:
Why did you remove the `rel`? The browser only infers `noopener`but not `noreferrer`